packages/main/src/components/ObjectPage/index.tsx

```diff
@@ -31,6 +31,7 @@ import { AvatarSize } from '@ui5/webcomponents-react/lib/AvatarSize';
 import { AvatarShape } from '@ui5/webcomponents-react/lib/AvatarShape';
 import { ContentDensity } from '@ui5/webcomponents-react/lib/ContentDensity';
 import '@ui5/webcomponents/dist/icons/navigation-up-arrow.js';
+import { getScrollBarWidth } from '@ui5/webcomponents-react-base';
```

Comment:
can you please import from
```suggestion
import { getScrollBarWidth } from '@ui5/webcomponents-react-base/lib/utils';
```
?